pkg/controller/daemon/daemon_controller.go

```diff
@@ -822,16 +822,16 @@ func (dsc *DaemonSetsController) podsShouldBeOnNode(
 				inBackoff := dsc.failedPodsBackoff.IsInBackOffSinceUpdate(backoffKey, now)
 				if inBackoff {
 					delay := dsc.failedPodsBackoff.Get(backoffKey)
-					klog.V(4).Infof("Deleting failed pod %s/%s on node %s has been limited by backoff - %v remaining",
-						pod.Namespace, pod.Name, node.Name, delay)
+					klog.V(4).InfoS("Deleting failed Pod on Node has been limited by backoff",
+						"pod", klog.KObj(pod), "node", klog.KObj(node), "delay", delay)
 					dsc.enqueueDaemonSetAfter(ds, delay)
 					continue
 				}
 
 				dsc.failedPodsBackoff.Next(backoffKey, now)
 
 				msg := fmt.Sprintf("Found failed daemon pod %s/%s on node %s, will try to kill it", pod.Namespace, pod.Name, node.Name)
-				klog.V(2).Infof(msg)
+				klog.V(2).InfoS(msg)
```

Comment:
this doesn't include pod/node info in structured form

Reply:
Fixed.